tests/test_slots.py

```diff
@@ -0,0 +1,26 @@
+from falcon import Request, Response
+import falcon.testing as testing
+
+
+class TestSlots(testing.TestBase):
+
+    def __init__(self):
+        self.failed = False
+        super(TestSlots, self).__init__()
+
+    def test_slots_request(self):
+        env = testing.create_environ()
+        req = Request(env)
+        try:
+            req.doesnt = 'exist'
+        except AttributeError:
+            self.failed = True
```

Comment:
if I change that up, then how should the test be updated as it is using self.fail

```
self.assertFalse(self.failed, 'Unable to add to __slots__ dynamically')
```

what about:

```
class TestSlots(testing.TestBase):

    def __init__(self):
        self.failed = False
        super(TestSlots, self).__init__()

    def test_slots_request(self):
        env = testing.create_environ()
        req = Request(env)
        try:
            req.doesnt = 'exist'
        except AttributeError:
            self.failed = True
        self.assertFalse(self.failed, self.fail('Unable to add to __slots__ dynamically'))

    def test_slots_response(self):
        resp = Response()
        try:
            resp.doesnt = 'exist'
        except AttributeError:
            self.failed = True
        self.assertFalse(self.failed, self.fail('Unable to add to __slots__ dynamically'))
```